packages/mongodb/packaging

```diff
@@ -7,7 +7,11 @@ set -u # report the usage of uninitialized variables
 
 export HOME=/var/vcap
 
-for i in `find mongodb/ -type f -name 'mongodb-linux-x86_64-*.tar.gz' -print`
+# retrieve the stemcell type
+
+typeset -l stemcell=$(grep -w "^ID" /etc/os-release|cut -d"=" -f2|tr -d \")
+
+for i in `find mongodb/ -type f -name "mongodb-${stemcell}-x86_64-*.tar.gz" -print`
```

Comment:
You should use the Bash syntax `$()` here instead of the old backquotes syntax, and add a `#!/usr/bin/env bash` sheebang header at the start of this script.

Also, you should not put a tailing slash `/` in the `find` first argument and just write `find mongodb -type f ...` because the trailing slash leads to double slashes in the resulting paths.